orca-api/src/main/java/com/netflix/spinnaker/orca/api/SimpleStageOutput.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.api;
+
+import com.google.common.annotations.Beta;
+import java.util.Map;
+
+@Beta
+public class SimpleStageOutput {
+  private SimpleStageStatus status;
+
+  public void setStatus(SimpleStageStatus status) {
+    this.status = status;
```

Comment:
Can you use `@Data` instead of adding getter/setter?

Reply:
We are trying not to add more dependencies than what we need. Right now we only have one dependency, which is for the `@Beta` annotation . With how small the classes are, we didn't think it it was an issue to create the getter/setters

Reply:
It's just for code readability. I'm fine not adding lombok as a dependency if it's not there already.

Reply:
Lombok is not a runtime dependency, it will be fine to add.